android/app/src/main/AndroidManifest.xml

```diff
@@ -59,16 +59,43 @@
           <data android:pathPrefix="/flagship" />
         </intent-filter>
       </activity>
+
+      <activity-alias
+        android:name="io.cozy.flagship.mobile.MainActivitycozy"
+        android:enabled="false"
+        android:exported="true"
+        android:icon="@mipmap/cozy"
+        android:targetActivity=".MainActivity">
+        <intent-filter>
+          <action android:name="android.intent.action.MAIN" />
+          <category android:name="android.intent.category.LAUNCHER" />
+        </intent-filter>
+      </activity-alias>
+
+      <activity-alias
+        android:name="io.cozy.flagship.mobile.MainActivitymespapiers"
+        android:enabled="false"
+        android:exported="true"
+        android:icon="@mipmap/mespapiers"
+        android:targetActivity=".MainActivity">
+        <intent-filter>
+          <action android:name="android.intent.action.MAIN" />
+          <category android:name="android.intent.category.LAUNCHER" />
+        </intent-filter>
+      </activity-alias>
+
       <activity
         android:name="com.zoontek.rnbootsplash.RNBootSplashActivity"
         android:theme="@style/BootTheme"
+        android:enabled="false"
```

Comment:
Sure about this one? Can you explain why?

Reply:
Sorry, it is because it generates duplicate icons. See https://github.com/skb1129/react-native-change-icon/issues/44#issuecomment-974042627